### PR TITLE
[BUGFIX] Revert changes made to dependencies in #4508

### PR DESCRIPTION
**Pull request by cdkini**
Changes proposed in this pull request:
- Revert changes made in #4508 due to various issues with missing dependencies breaking the build.

### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have run any local integration tests and made sure that nothing is broken.

### CODE REVIEW

File: great_expectations/data_context/util.py [hunk 1]
```diff
@@ -542,39 +542,46 @@ def mask_db_url(cls, url: str, use_urlparse: bool = False, **kwargs) -> str:
             url with password masked e.g. "postgresql+psycopg2://username:***@host:65432/database"
         """
         if sa is not None and use_urlparse is False:
-            engine = sa.create_engine(url, **kwargs)
-            return engine.url.__repr__()
+            try:
+                engine = sa.create_engine(url, **kwargs)
+                return engine.url.__repr__()
+            # Account for the edge case where we have SQLAlchemy in our env but haven't installed the appropriate dialect to match the input URL
+            except Exception as e:
```
Comment (cdkini) [inline great_expectations/data_context/util.py]: Test changes from another branch - once this full build passes, I know that the hotfix has worked as intended. This is not to be merged with these changes.

Comment (cdkini) [inline great_expectations/data_context/util.py]: Test changes from another branch - once this full build passes, I know that the hotfix has worked as intended. This is not to be merged with these changes.

**Review by talagluck — APPROVED** [review]
LGTM

**Review by austiezr — APPROVED** [review]
Breaks my heart /s 

but also LGTM!

**Pull request merged**